docs/source/admin/quick_howto/ciab.rst

```diff
@@ -0,0 +1,115 @@
+	+---------------------------------+--------------------------------------------------------------+---------------------------------------+-------------------------------------------+
+	| Traffic Ops PostgresQL Database | PostgresQL connections accepted on port 5432 (database name: | ``DB_USER`` in `variables.env`_       | ``DB_USER_PASS`` in `variables.env`_      |
+	|                                 | ``DB_NAME`` in `variables.env`_)                             |                                       |                                           |
+	+---------------------------------+--------------------------------------------------------------+---------------------------------------+-------------------------------------------+
+	| Traffic Portal                  | Web interface on 443 (Javascript required)                   | ``TO_ADMIN_USER`` in `variables.env`_ | ``TO_ADMIN_PASSWORD`` in `variables.env`_ |
+	+---------------------------------+--------------------------------------------------------------+---------------------------------------+-------------------------------------------+
+	| Traffic Router                  | Web interfaces on ports 3080 (HTTP) and 3443 (HTTPS), with a | N/A                                   | N/A                                       |
+	|                                 | DNS service on 53 and an API on 3333                         |                                       |                                           |
+	+---------------------------------+--------------------------------------------------------------+---------------------------------------+-------------------------------------------+
+	| Traffic Vault                   | Riak key-value store on port 8010                            | ``TV_ADMIN_USER`` in `variables.env`_ | ``TV_ADMIN_PASSWORD`` in `variables.env`_ |
+	+---------------------------------+--------------------------------------------------------------+---------------------------------------+-------------------------------------------+
+
+.. seealso:: :ref:`tr-api` and :ref:`tm-api`
+
+While the components may be interacted with by the host using these ports, the true operation of the CDN can only truly be seen from within the Docker network. To see the CDN in action, connect to a container within the CDN in a Box project and use cURL to request the URL ``http://video.demo1.mycdn.ciab.test`` which will be resolved by the DNS container to the IP of the Traffic Router, which will provide a ``302 FOUND`` response pointing to the Edge-Tier cache. A typical choice for this is the "enroller" service, which has a very nuanced purpose not discussed here but already has the ``curl`` command line tool installed. For a more user-friendly interface into the CDN network, see `The Test Client`_.
+
+.. code-block:: shell
+	:caption: Example Command to See the CDN in Action
+
+	sudo docker exec -it cdninabox_enroller_1 /usr/bin/curl -L "http://video.demo1.mycdn.ciab.test"
```

Comment:
`docker-compose exec enroller /usr/bin/curl -L http://video.demo1.mycdn.ciab.test` will accomplish the same and is a bit easier to digest.

Reply:
I actually didn't know that. I'm thinking I should keep the `sudo`, because most default Docker installs don't automatically set any particular user's permissions such that they are able to access the `dockerd` without it